content/03.text-mining.md

```diff
@@ -12,9 +12,37 @@
 
 ### Text Mining for Relationship Extraction
 
-#### Rule-Based Natural Language Processing
+#### Rule-Based Relationship Extraction
 
-1. Mention papers on hand written rules and expressions
+Rule-based extraction consists of identifying sentences that contain important keywords or grammatical patterns that allude to relationships of interest. 
+These keywords are established via expert knowledge or pre-existing ontologies.
```

Comment:
Might be good to revise this, since the previous sentence has an `or`. I think you could probably drop `These`